lib/subscription.rb

```diff
@@ -0,0 +1,23 @@
+class Subscription
+  def self.keyname(service)
+    "claim_review_webhooks_#{service}"
+  end
+
+  def self.add_subscription(service, url)
+    REDIS_CLIENT.sadd(self.keyname(service), url)
```

Comment:
What happens here when `service` is an array?

Reply:
not a terrible idea - I'm going to make these transform `service` and `url` into arrays and loop through both so you can send either arrays or strings through.